cobigen-cli/src/main/java/cobigen.bat

```diff
@@ -0,0 +1,21 @@
+ECHO OFF
+set path=C:\Program Files\Java\jdk1.8.0_181\bin
+set CLASSPATH=.;C:\Program Files\Java\jdk1.8.0_181;C:\Program Files\Java\jdk1.8.0_181\lib\dt.jar;C:\MyData\IDE4\workspaces\com.cobigen.picocli\picocli-2.2.1.jar;C:\Users\syadav9\.m2\repository\com\devonfw\cobigen\openapiplugin\2.2.0\openapiplugin-2.2.0.jar
```

Comment:
We need to find a way to use paths that does not depend on your computer, like relative paths. 

Also, for the Java installation, most probably we need something similar to what the devonfw-distribution does. If you go to the file `/scripts/environment-project-bat` you will see something like this:

```
set JAVA_HOME=%SOFTWARE_PATH%\java
```

However, I'm not pretty sure about this solution. What do you think @maybeec ?